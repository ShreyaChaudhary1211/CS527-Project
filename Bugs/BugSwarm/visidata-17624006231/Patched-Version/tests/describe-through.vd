sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/benchmark.csv	o	
benchmark	Customer		select-col-regex	Robert	|	
benchmark	Quantity		setcol-expr	None	g=	
benchmark	Quantity		type-int		#	
benchmark			describe-sheet		I	
benchmark_describe	nulls	ã‚­Quantity	dive-cell		z^J	
benchmark_Quantity_nulls	Unit		type-currency		$	
benchmark_Quantity_nulls	Paid		type-currency		$	
