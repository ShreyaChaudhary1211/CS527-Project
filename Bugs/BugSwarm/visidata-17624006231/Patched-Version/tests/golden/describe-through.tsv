Date	Customer	SKU	Item	Quantity	Unit	Paid
7/3/2018 1:47p	Robert Armstrong	FOOD213	BFF Oh My Gravy! Beef & Salmon 2.8oz		12.95	51.80
7/13/2018 3:49p	Robert Armstrong	FOOD216	BFF Oh My Gravy! Chicken & Shrimp 2.8oz		12.95	51.80
7/17/2018 9:01a	Robert Armstrong	FOOD217	BFF Oh My Gravy! Duck & Tuna 2.8oz		12.95	51.80
7/23/2018 1:41p	Robert Armstrong	FOOD215	BFF Oh My Gravy! Lamb & Tuna 2.8oz		12.95	51.80
8/6/2018 10:21a	Robert Armstrong	FOOD212	BFF Oh My Gravy! Beef & Chicken 2.8oz		12.95	51.80
8/10/2018 4:31p	Robert Armstrong	FOOD211	BFF Oh My Gravy! Chicken & Turkey 2.8oz		12.95	51.80
8/21/2018 12:13p	Robert Armstrong	FOOD214	BFF Oh My Gravy! Duck & Salmon 2.8oz		12.95	51.80
8/24/2018 11:42a	Robert Armstrong	FOOD218	BFF Oh My Gravy! Chicken & Salmon 2.8oz		12.95	51.80
8/29/2018 10:07a	Robert Armstrong	FOOD219	BFF Oh My Gravy! Chicken & Pumpkin 2.8oz		12.95	51.80
8/31/2018 12:00a	Robert Armstrong	FOOD219	BFF Oh My Gravy! Chicken & Pumpkin 2.8oz		12.95	1864.80
